pkg/virt-handler/device-manager/usb/manager_test.go

```diff
@@ -145,3 +257,8 @@ func nodeConfigHelper(name string, devices []string) *v1alpha1.NodeConfig {
 	}
 	return config
 }
+
+func toInt(str string) int {
+	val, _ := strconv.ParseInt(str, 16, 32)
```

Comment:
Please, check the error too, you can simply add `Expect(err).ShouldNot(HaveOccurred())`